packages/blade/src/components/Radio/useRadio.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import type { GestureResponderEvent } from 'react-native';
```

Comment:
Importing from `react-native` in a file that can be used on web is dangerous. This will cause failures on projects that are react only and wouldn't be able to find `react-native`. Lets not do this?

Reply:
It's a `type` import.

Reply:
But it's coming from `react-native` package which won't exist on web only projects

Reply:
I'm in favor of having two different modules .web, .native for this. Thoughts?

Reply:
> But it's coming from react-native package which won't exist on web only projects

type imports are stripped away and won't even be imported/bundled on the web project. 

Anyway we will have this on each and every component when we have TS 4.7 module suffixes.

Reply:
Since its only type and its going to be stripped off, I think its okay. With TS 4.7 we'll see what to do